### ISSUE TITLE
Generate 0 predictions in reproducing the results

### ISSUE THREAD

**Issue by thss15fyt** [bfshi/DGAM-Weakly-Supervised-Action-Localization#11]
Hi, thanks for your contribution to this interesting work.

I have downloaded the datasets and set `config.DATASET_NAME = 'ActivityNet12'` to reproduce the results. However, during the training process, I got the following output with an averaged-mAP of 0.0:

Number of predictions: 0
Warning: No predictions of label XXX were provided.

I do not modify any code of this library. So is there anything missed for reproducing your results? Thanks.

**Comment by bfshi** [comment]
Hi, thanks for your interest in our work.

In your case, do you mean there are no predictions for any label? or just for one label? 

And is the loss decreasing in training? Is there anything strange in training except for the training-time evaluation?

**Comment by thss15fyt** [comment]
Yes, there is no prediction for any label. Actually, I print out the classification score and find all frames are predicted as the background label.

The loss is continually decreasing from 5 to 0.2, after training 500+ epochs. However, the predictions keep empty throughout the time.

**Comment by bfshi** [comment]
I just downloaded the code and data, and tried as you said (only change config.DATASET_NAME and nothing else). Everything seems normal. Only in the first 10 epochs there are no predictions, but after that I get positive mAP.

Could you try to change `config.IF_DETERMINISTIC` to False, and run the code again?

**Comment by thss15fyt** [comment]
@bfshi It does not work to set the flag to False. Is it possible that some packages are installed in different versions from the `requirements.txt`?

**Comment by bfshi** [comment]
I just double-checked and seems it's the same version on my server...
